videolibrary_get.go

```diff
@@ -0,0 +1,80 @@
+	WatermarkWidth        *int32  `json:"WatermarkWidth,omitempty"`
+	PullZoneID            *int64  `json:"PullZoneId,omitempty"`
+	StorageZoneID         *int64  `json:"StorageZoneId,omitempty"`
+	WatermarkHeight       *int32  `json:"WatermarkHeight,omitempty"`
+	EnabledResolutions    *string `json:"EnabledResolutions,omitempty"`
+
+	ViAiPublisherID                  *string  `json:"ViAiPublisherId,omitempty"`
+	VastTagURL                       *string  `json:"VastTagUrl,omitempty"`
+	WebhookURL                       *string  `json:"WebhookUrl,omitempty"`
+	CaptionsFontSize                 *int32   `json:"CaptionsFontSize,omitempty"`
+	CaptionsFontColor                *string  `json:"CaptionsFontColor,omitempty"`
+	CaptionsBackground               *string  `json:"CaptionsBackground,omitempty"`
+	UILanguage                       *string  `json:"UILanguage,omitempty"`
+	AllowEarlyPlay                   *bool    `json:"AllowEarlyPlay,omitempty"`
+	PlayerTokenAuthenticationEnabled *bool    `json:"PlayerTokenAuthenticationEnabled,omitempty"`
+	AllowedReferrers                 []string `json:"AllowedReferrers,omitempty"`
+	BlockedReferrers                 []string `json:"BlockedReferrers,omitempty"`
+	BlockNoneReferrer                *bool    `json:"BlockNoneReferrer,omitempty"`
+	EnableMP4Fallback                *bool    `json:"EnableMP4Fallback,omitempty"`
+	KeepOriginalFiles                *bool    `json:"KeepOriginalFiles,omitempty"`
+	AllowDirectPlay                  *bool    `json:"AllowDirectPlay,omitempty"`
+	EnableDRM                        *bool    `json:"EnableDRM,omitempty"`
+	Bitrate240p                      *int32   `json:"Bitrate240p,omitempty"`
+	Bitrate360p                      *int32   `json:"Bitrate360p,omitempty"`
+	Bitrate480p                      *int32   `json:"Bitrate480p,omitempty"`
+	Bitrate720p                      *int32   `json:"Bitrate720p,omitempty"`
+	Bitrate1080p                     *int32   `json:"Bitrate1080p,omitempty"`
+	Bitrate1440p                     *int32   `json:"Bitrate1440p,omitempty"`
+	Bitrate2160p                     *int32   `json:"Bitrate2160p,omitempty"`
+	APIAccessKey                     *string  `json:"ApiAccessKey,omitempty"`
+	ShowHeatmap                      *bool    `json:"ShowHeatmap,omitempty"`
+	EnableContentTagging             *bool    `json:"EnableContentTagging,omitempty"`
+	PullZoneType                     *int32   `json:"PullZoneType,omitempty"`
+	CustomHTML                       *string  `json:"CustomHTML,omitempty"`
+	Controls                         *string  `json:"Controls,omitempty"`
+	PlayerKeyColor                   *string  `json:"PlayerKeyColor,omitempty"`
+	FontFamily                       *string  `json:"FontFamily,omitempty"`
+}
+
+// VideoLibraryGetOpts represents optional query parameters available when Getting or Listing Video Libraries
+type VideoLibraryGetOpts struct {
+	IncludeAccessKey bool `url:"includeAccessKey,omitempty"`
```

Comment:
What do you think about removing `omitempty`? 
Currently if IncludeAccessKey is false, it would not be passed as request parameter.
If the behavior on bunny.net would change regarding if a missing `includeAccessKey` parameter means true or false, the behaviour of the bunny-go lib would change too.

By removing `omitempty` we can guarantee consistent behaviour, independent of a change on bunny.net side regarding the default for a missing includeAccessKey parameter.